### PR TITLE
feat: suite desktop download landing page

### LINKED ISSUE #2324: New landing page
Destination: suite.trezor.io

Design: [Zeplin](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/5f7589dc8b91544e3e52c98b)

### PR DESCRIPTION
**Pull request by slowbackspace**
@tomexx will take it from there and implement the page itself

### CODE REVIEW

**Comment by prusnak** [comment]
We need to update the assetPrefix and s3sync scripts similarly to what I've done in https://github.com/trezor/trezor-suite/pull/2459.

Let's merge this PR first (once it is done) and then I'll update https://github.com/trezor/trezor-suite/pull/2459 to fix stuff in `suite-landing-page` as well.

**Comment by slowbackspace** [comment]
@tomexx lint and type-check are failing, you can run  run tests locally with `yarn workspace @trezor/suite-web-landing lint/type-check`

**Comment by tomexx** [comment]
It's still WIP, I'll put more updates over the weekend and then clean up the linting issues. Thanks

**Comment by tomexx** [comment]
The LP is ready for review but there will be a new copy added later once provided by @matejzak

File: packages/suite-web-landing/components/Download/index.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+];
+
+const getIconForCurrentPlatform = (platform: Platform) => {
+    return dropdownItemsData.find(item => platform === item.platform)!.icon;
+};
+
+const getInstallerURI = (platform: Platform, version: string) => {
+    const extension = dropdownItemsData.find(item => platform === item.platform)!
+        .installerExtension;
+    return encodeURI(`/wallet/static/desktop/Trezor Suite-${version}.${extension}`);
```
Comment (slowbackspace) [inline packages/suite-web-landing/components/Download/index.tsx:95]: just a reminder to make sure if these are indeed correct paths

File: packages/suite-web-landing/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "dev": "yarn workspace @trezor/suite-data copy-static-files && next",
+        "build:app": "next build && next export -o build",
+        "build": "run-s copy-files build:app"
+    },
+    "dependencies": {
+        "@trezor/components": "1.0.0",
+        "next": "^9.5.3",
+        "react": "16.13.1",
+        "react-dom": "16.13.1",
+        "react-reveal": "^1.2.2"
```
Comment (slowbackspace) [inline packages/suite-web-landing/package.json]: not sure if we shouldn't switch to something that is till supported, but I guess it's not a priority right now 
https://github.com/rnosov/react-reveal/issues/106

Comment (tomexx) [reply packages/suite-web-landing/package.json]: I think I can switch it to `react-awesome-reveal` which they recommend.

File: packages/suite-web-landing/components/Download/index.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+};
+
+const Index = () => {
+    const version: string = process.env.VERSION ? normalizeVersion(process.env.VERSION) : '';
+    const [platform, setPlatform] = useState<Platform>('linux');
+    const dropdownItems = dropdownItemsData.map(item => ({
+        key: 'items',
+        options: [
+            {
+                key: item.platform as string,
```
Comment (slowbackspace) [inline packages/suite-web-landing/components/Download/index.tsx]: total nitpick, I feel like excluding undefined from DropdownItem.platform type would be a bit nicer than this type assertion

Comment (tomexx) [reply packages/suite-web-landing/components/Download/index.tsx]: > total nitpick, I feel like excluding undefined from DropdownItem.platform type would be a bit nicer than this type assertion

Yes, I have actually already excluded that, I believe. I'll just remove this. Good catch :)

File: packages/suite-web-landing/package.json (hunk 2 above)
Comment (tomexx) [reply packages/suite-web-landing/package.json]: Done, thanks

File: packages/suite-web-landing/components/Layout/index.tsx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+        <Footer>
+            <FooterLinks>
+                <FooterList>
+                    <FooterHeadline>Improve</FooterHeadline>
+                    <FooterLink href="https://www.typeform.com/private-typeform">
+                        Give feedback
+                    </FooterLink>
+                    <FooterLink href="https://blog.trezor.io/join-the-trezor-beta-testers-community-b19761f4960a">
+                        Join closed Beta
+                    </FooterLink>
```
Comment (keraf) [inline packages/suite-web-landing/components/Layout/index.tsx:134]: Is there still going to be a closed beta after this release @matejzak ?

Comment (glacierbits) [reply packages/suite-web-landing/components/Layout/index.tsx:134]: Yes, sir. @keraf ("Buy" feature will be release to closed beta upon QA's approval. Planned for next week).

File: packages/suite-web-landing/pages/index.tsx [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+    color: ${colors.NEUE_TYPE_ORANGE};
+`;
+
+const features = [
+    {
+        id: 1,
+        headline: 'Desktop app. 2x safer.',
+        text:
+            'Security and privacy. Anti-Phishing, Bitcoin\nCore, Tor support, Discreet mode, Performance.',
+        backgroundPosition: 'bottom right',
```
Comment (keraf) [inline packages/suite-web-landing/pages/index.tsx:100]: We don't have Bitcoin Core and Tor support atm, should be switch the soon boolean on? cc @matejzak

File: packages/suite-web-landing/pages/index.tsx [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+                </Fade>
+                <DownloadWrapper>
+                    <Fade delay={2000} triggerOnce>
+                        <Download />
+                    </Fade>
+                </DownloadWrapper>
+            </StyledHeroCta>
+            <FeaturesWrapper>
+                {features.map((item, key) => {
+                    return (
```
Comment (keraf) [inline packages/suite-web-landing/pages/index.tsx]: Nitpicking here but the explicit return statement isn't necessary.

**Review by keraf — APPROVED** [review]
Looks good to me!

File: packages/suite-web-landing/pages/index.tsx (hunk 5 above)
Comment (glacierbits) [reply packages/suite-web-landing/pages/index.tsx:100]: Those are just copy placeholders and are there to illustrate the direction of that particular paragraph. Copy is to be updated after the freeze.

File: packages/components/src/images/trezor_logo_suite.svg [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
Comment (szymonlesisz) [inline packages/components/src/images/trezor_logo_suite.svg]: is this doctype tag in svg necessary? (just curious, i didnt see that in other svg files and doesnt have a close tag)

File: packages/suite-web-landing/components/Layout/index.tsx [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+    <Layout>
+        <Header>
+            <StyledTrezorLogo type="suite" width={185} />
+            <Button
+                variant="tertiary"
+                icon="EXTERNAL_LINK"
+                alignIcon="right"
+                color={colors.NEUE_TYPE_DARK_GREY}
+            >
+                <Link variant="nostyle" href="https://suite.trezor.io/web">
```
Comment (szymonlesisz) [inline packages/suite-web-landing/components/Layout/index.tsx]: should it be hardcoded? what about usage in stage/sldev env?

File: packages/suite-web-landing/next-env.d.ts [hunk 9]
```diff
@@ -0,0 +1,3 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
+/// <reference path="../suite/global.d.ts" />
```
Comment (szymonlesisz) [inline packages/suite-web-landing/next-env.d.ts]: global.d.ts are not used in this project (desktopApi, redux dev tools)

**Review by szymonlesisz — CHANGES_REQUESTED** [review]

File: packages/components/src/images/trezor_logo_suite.svg (hunk 7 above)
Comment (tomexx) [reply packages/components/src/images/trezor_logo_suite.svg]: probably we can get rid of that, I'll run it through SVG optimizer. Thanks

File: packages/suite-web-landing/components/Layout/index.tsx (hunk 8 above)
Comment (prusnak) [reply packages/suite-web-landing/components/Layout/index.tsx]: If there is no better suggestion, let's use this:
```suggestion
                <Link variant="nostyle" href="/web">
```

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**